handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -410,14 +410,12 @@ private static void loadTcNative() throws Exception {
         Set<String> libNames = new LinkedHashSet<String>(4);
         // First, try loading the platform-specific library. Platform-specific
         // libraries will be available if using a tcnative uber jar.
-        libNames.add("netty-tcnative-" + os + '-' + arch);
+        libNames.add("netty_tcnative-" + os + '-' + arch);
         if (LINUX.equalsIgnoreCase(os)) {
             // Fedora SSL lib so naming (libssl.so.10 vs libssl.so.1.0.0)..
-            libNames.add("netty-tcnative-" + os + '-' + arch + "-fedora");
+            libNames.add("netty_tcnative-" + os + '-' + arch + "-fedora");
         }
         // finally the default library.
-        libNames.add("netty-tcnative");
-        // in Java 8, statically compiled JNI code is namespaced
         libNames.add("netty_tcnative");
 
         NativeLibraryLoader.loadFirstAvailable(SSL.class.getClassLoader(),
```

Comment:
actually I think we can just not support the use case where netty-tcnative and netty have different package prefixes if we are assuming netty-tcnative will move into the netty project soon.

If we do want to support different package prefix (or for example one shaded and the other one not) then we will have to use the native library loading from `Library` in netty-tcnative (which would be able to infer the package prefix), as opposed to `NativeLibraryLoader` which lives in Netty and cannot as easily infer the package prefix for classes in another jar.